easybuild/easyblocks/o/openfoam.py

```diff
@@ -292,6 +292,25 @@ def configure_step(self):
                         env.setvar("BOOST_ROOT", get_software_root('Boost'))
                     else:
                         env.setvar("%s_ROOT" % depend.upper(), dependloc)
+            for dep in self.cfg.dependencies():
+                # CGAL >= 5.x is header-only, but when using it OpenFOAM still needs MPFR and GMP.
```

Comment:
comment mentions GMP, but nothing is being done for GMP below, only MPFR?

Reply:
Yeah, good question. It does look like it needs GMP, but there's no `GMP_LIB_DIR`... In for instance https://develop.openfoam.com/Development/openfoam/-/blob/OpenFOAM-v2312/wmake/rules/General/cgal-header-only?ref_type=tags they do add `-lgmp`. So I'm not completely sure what to do here...

Reply:
With the current approach it does build fine, but we did have undefined references to GMP before. So I guess it does somehow find GMP.